### PR TITLE
Add a reference to step by step installation exmple on wiki page

### PR DESCRIPTION
**Pull request by vrainish-pacbio**

### CODE REVIEW

File: README.INSTALL.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Installation
 
+### See a step by step Blasr installation example on Blasr wiki page
```
Comment (pb-cdunn) [inline README.INSTALL.md]: This should include the actual URL. E.g.
* https://github.com/PacificBiosciences/blasr/wiki/step-by-step-blasr-installation-example

**Comment by pb-cdunn** [comment]
Just push a new commit to your current `master` branch (the one you used for this pull-request) in your personal fork, **vrainish-pacbio:master** (the source repo for this pull-request).

E.g. if your personal remote (the alias for your fork) is called `origin`, then:

    # modify code
    git add .
    git commit -m 'my message'
    git push origin master

Btw, I've updated your directions to make them copyable and directly runnable:
* https://github.com/PacificBiosciences/blasr/wiki/Step-by-step-blasr-installation-example

**Comment by vrainish-pacbio** [comment]
Hi Chris,

Thanks for your response. Another minor question.
I pushed a small completely independent change ( version number in consize help) and wanted to create a new pull request but in the menu it also include changes to INSTALL.
How to create two separate full requests for two files pushed ? (sorry to nag you with minor things)

Regards, Vladimir

**Comment by pb-cdunn** [comment]
Two separate pull requests. That is *precisely* the reason for feature branches. So I'm not answering this question. You need to start using branches, now that you understand the use-case.

**Pull request closed without merging**